tests/test_routes_segments.py

```diff
@@ -0,0 +1,250 @@
+import datetime
+import unittest
+import datamodels
+from app import app
+
+from .utils import make_authorized_call
+
+
+class TestSegmentsRoutes(unittest.TestCase):
+    def setUp(self):
+        with app.app_context():
+            self.session = datamodels.get_session()
+
+    def tearDown(self):
+        datamodels._clear_session_for_tests()
+
+    @staticmethod
```

Comment:
Why are these static methods?  They should only be used within this test class, which will be initialized while the tests are running.